src/components/pages/document.js

```diff
@@ -1097,6 +1109,23 @@ function Document({ initialState, PageTemplate }) {
     }
   };
 
+  const renderDocumentRating = () => {
+    const starRating = (
```

Comment:
Small `starRating` naming leftover here.

Reply:
Nice spot, I renamed it (preliminary, as it might get renamed once more with https://educandu.atlassian.net/browse/EDU-1551).